ansible/playbook.yml

```diff
@@ -2,4 +2,4 @@
 - hosts: artifactory
   become: true
   roles:
-    - artifactory
+    - role: artifactory
```

Comment:
We tend to use either the approach you had previously:
`- artifactory` or wrap with curly braces like this `- { role: artifactory }`